internal/kibana/connector.go

```diff
@@ -198,6 +197,26 @@ func resourceConnectorRead(ctx context.Context, d *schema.ResourceData, meta int
 	return flattenActionConnector(connector, d)
 }
 
+func resourceConnectorsRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
This looks like it's only used by the data source now. Can we move it into the ds file and rename it to datasourceConnectorRead or something.

Reply:
sure